### DISCUSSION TITLE
In the grouping of tables, border lines of empty rows will appear

### DISCUSSION THREAD

**Discussion by fadeli1** [eclipse-birt/birt#1259] [Q&A] [answered]
word actual result
![image](https://user-images.githubusercontent.com/1914030/230886148-94ddc3f0-af6d-4ad0-869b-0be98c64d4d0.png)

word hope result
![image](https://user-images.githubusercontent.com/1914030/230890115-01ba4af1-1f35-48e8-9c9e-f735bebd334d.png)

birt version: 4.9.0

is there a way to achieve this?

[new_report.zip](https://github.com/eclipse/birt/files/11190001/new_report.zip)

**Comment by speckyspooky** [comment]
This problem isn't a Word-problem and this isn't an issue of BIRT this is standard behavior.
And the solution of the space is given into 2 changes on your layout.
You will get the space due to the stanard padding of the cells and that your have drawn the cell-lines on each row.
Step 1 is to remove the padding (set to 0) and the second steps is to remove the top-line of the detail cells because your use the group-merge and so the grouping line will draw your above line.

Here is the result of my change based on your table with HTML, DOCX & PDF:
![2023-04-10 14_09_58-Table](https://user-images.githubusercontent.com/41593722/230899798-f7140f1d-d2bb-40e9-9328-56e083154a58.png)

Here you get a zip file with my change and the 2 result examples:
[new_report_changed_speckyspooky.zip](https://github.com/eclipse/birt/files/11190398/new_report_changed_speckyspooky.zip)

**Reply by fadeli1** [reply]
when there is only one group, there is no problem according to the settings. When there are multiple groups, there is an obvious gap in the border line
![image](https://user-images.githubusercontent.com/1914030/231070082-799f6ca1-3750-4864-9501-c7eb04d6cad8.png)
[new_report.zip](https://github.com/eclipse/birt/files/11197136/new_report.zip)

**Reply by speckyspooky** [reply, marked as answer]
The solution in this case is to add the border on the left & right side of the cell which haven't currently borders.
My screen shows you my designer with the new borders and the DOCX-result.

![2023-04-12 21_06_03-workspace - C__Users_gutma_Downloads_new_report_2_new_report_2 rptdesign - BIRT ](https://user-images.githubusercontent.com/41593722/231559711-1014f8fd-af0d-4a6b-b505-b0cdf3124f2f.png)

**Reply by fadeli1** [reply]
thank you

**Reply by fadeli1** [reply]
@speckyspooky 
I have other questions, hope you can help me answer

I use birt-report-designer-all-in-one-4.9.0-20220315-win32.win32.x86_64.zip

I use POJO Data source

pojo.jar code
```
public class test {
    
    public void open(Object appContext, Map<String,Object> map) {
        System.out.println(appContext);
        System.out.println(map);
    }
    
    public Object next() {
        
    }
    
    public void close() {
    }
}
```
question 1: 
I want to output the corresponding information when I can check it through the birt viewer of eclipse, which is convenient for debugging
![image](https://user-images.githubusercontent.com/1914030/233012267-399ff140-7e25-4b09-80ba-328f180b8fcc.png)

question 2: 
after I recompile the pojo.jar package, I have to close eclipse, reopen eclipse, click birt viewer again to see the latest result

**Comment by hvbtup** [comment]
Please do not "hijack" existing issues/discussions for unrelated questions.

**Reply by fadeli1** [reply]
sorry
